demos/sev-demo/README.md

```diff
@@ -0,0 +1,136 @@
+# CCv0 with AMD SEV
+
+[Demo Video](https://youtu.be/SXZ-hzhjahU)
+
+AMD SEV is already enabled in upstream Kata. SEV can also be used with CCv0. Although most of the code needed for CCv0 with SEV is upstream in CCv0, currently some manual configuration is required. This is described here. Some of these steps will soon be automated. These instructions assume that you already have CCv0 setup and that you have a node where you can launch SEV VMs.
+
+## OVMF
+
+You will need to use OVMF. Furthemore, you will need to use the AMDSEV build of OVMF. An appropriate firmware binary is available in this directory. You can also build from source. To do so, follow the standard OVMF build procedure, but use `edk2/OvmfPkg/AmdSev/AmdSevX64.dsc` rather than the OVMF DSC file. The AMDSEV build currently expects to use Grub for reasons unrelated to Kata. To bypass this, you should comment out line 28 of `edk2/OvmfPkg/AmdSev/AmdSevX64.dsc`. If the build process complains about a missing `grub.efi` file, you can create a dummy file with that name and try again. This dependency may be adjusted in the future and we are hoping to provide a prebuilt version of the appropriate firmware.
+
+## Rootfs
+ 
+Create the rootfs as you normally would for CCv0, but be sure to specify set `AA_KBC` to `offline_sev_kbc` so that the AA is built with the correct key broker client. 
```

Comment:
Maybe this would be slightly clearer?:

> but set the key broker client option `AA_KBC=offline_sev_kbc` so that the AA is built with the correct client.

This is a slightly awkward sentence, but it raises a related important issue. There are a lot of acronyms and abbreviations involved in CC, so it would be worth us coming up with a usage standard. Ideally, every document would introduce each term and define it on first use, and then use the shortened term from then on. But is that realistic to do in every document, given the sheer number of terms we're referring to? I'd say no. It needs discussion but maybe a better approach is always to use the acronymn/abbreviation but have a very prominent (and well maintained!) project glossary.

For example the sentence above specifies "key broker client" (the KBC), but that acronym hasn't be used yet; it's only been indirectly mentioned via the `AA_KBC` setting name.

ftr I've also raised https://github.com/confidential-containers/documentation/issues/13.

Reply:
I would be happy to work on an overall project glossary. I'll try to be a bit more clear in this file as well although the guide isn't really mean for people who don't know what a KBC is.